post-orientation-exercises/postgres/03-sequelize-models.md

```diff
@@ -51,12 +51,13 @@ Now that `associate` has been added as a property of a Beach, what the heck is i
 
 Beach.associate = function(models) {
   Beach.hasMany(models.Lifeguard, {
-    foreignKey: 'lifeguardId'
+    foreignKey: 'beach_id' //the same foreign key must be specified when you define the association on the lifeguard model.
   });
 };
 
 ...
 ```
+(When naming `foreignKey`s in your models, think of the 'hasMany' as sending the foreignKey, and the 'belongsTo' as recieving it.)
```

Comment:
Oh, I guess that answers my question. Weird. I guess I got that all wrong. 
Typo: "receiving"

Reply:
That's not a typo, that's how I spell the r-word that means 'to accept something given or sent' ✉️